### DISCUSSION TITLE
Scales to DX7? (What Yamaha DXs support microtuning)

### DISCUSSION THREAD

**Discussion by MeblIkea** [asb2m10/dexed#440] [General]
Hello!
I was wandering if it would be possible to be able to send scales / microtonal scales to the DX7.
I saw that in the settings it's possible to change the scale, but it only applies to the VST, while I would like to export it to the keyboard.
Is it possible pls?
Thanks!

**Comment by asb2m10** [comment]
No the original DX7 doesn't support microtuning but from what I've heard the DX7II does.
